xwiki-commons-core/xwiki-commons-properties/src/main/java/org/xwiki/properties/internal/DefaultBeanDescriptor.java

```diff
@@ -286,6 +286,22 @@ private void setCommonProperties(DefaultPropertyDescriptor desc, Map<Class, Anno
             }
             group.setFeature(parameterFeature.value());
         }
+
+
+        PropertyDisplayType displayTypeAnnotation = (PropertyDisplayType) annotations.get(PropertyDisplayType.class);
+        Type displayType;
+        if (displayTypeAnnotation != null && displayTypeAnnotation.value().length > 0) {
```

Comment:
If there is only one Class in the table you should use it directly. Creating a DefaultParameterizedType with no arguments would not make much sense.

Reply:
this is the case.

Reply:
Indeed I looked too quickly.